tests/foreman/api/test_reporttemplates.py

```diff
@@ -685,11 +685,12 @@ def test_positive_generate_job_report(setup_content, target_sat, rhel7_contentho
     :customerscenario: true
     """
     ak, org = setup_content
-    rhel7_contenthost.install_katello_ca(target_sat)
-    rhel7_contenthost.register_contenthost(org.label, ak.name)
-    rhel7_contenthost.add_rex_key(target_sat)
-    assert rhel7_contenthost.subscribed
-    # Run a Job on the Host
+    for host in content_hosts:
+        host.install_katello_ca(target_sat)
+        host.register_contenthost(org.label, ak.name)
```

Comment:
Would global registration fit here?
```suggestion
        host.register(org, None, ak.name, module_target_sat)
```

Reply:
@ColeHiggins2 Based on the latest discussion in _Automation Weekly BS_ call, this became rather a blocker than suggestion. Can you update the registering method?